### PR TITLE
MEPTS 1384 | TX RTT Difference in FGH DB

### PR DESCRIPTION
**Pull request by TinoMuianga**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxRttCohortQueries.java [hunk 1]
```diff
@@ -1139,4 +1142,58 @@ public CohortDefinition getPatientWithUnknownCd4Result() {
...
+   *
+   * <p>The system will consider patient as transferred out as above defined only if the most recent
+   * date between (next scheduled ART pick-up on FILA + 1 day) and (the most recent ART pickup date
+   * on Ficha Recepção – Levantou ARVs + 31 days) falls during the reporting period.
+   *
+   * </blockquote>
+   *
+   * @return {@link CohortDefinition}
+   */
+  public CohortDefinition getTrfOut() {
```
Comment (eliazmuchave) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxRttCohortQueries.java]: Use Java convention for naming methods

**Review by eliazmuchave — CHANGES_REQUESTED** [review]
Left comment. Take a look

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxRttCohortQueries.java (hunk 1 above)
Comment (TinoMuianga) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxRttCohortQueries.java]: Done

**Review by eliazmuchave — APPROVED** [review]
Looks good to me

**Pull request merged**